lib/build-metadata.ts

```diff
@@ -37,10 +39,10 @@ export class BuildMetadata {
 			const buffer = await TarUtils.streamToBuffer(stream);
 			const relative = this.getMetadataRelativePath(header.name);
 
-			if (relative != null) {
-				this.addMetadataFile(relative, buffer);
-			} else {
+			if (relative == null || relative === QEMU_BIN_NAME) {
 				pack.entry(header, buffer);
+			} else {
+				this.addMetadataFile(relative, buffer);
```

Comment:
I've added a commit to this PR to solve an error I came across while testing the "balena build" command:
```
[Error]   Build failed
Error: COPY failed: stat /var/lib/docker/tmp/docker-builder263845279/.balena/qemu-execve: no such file or directory
```
I reckon that the "metadata extraction" code in resin-multibuild was deleting all contents under the `.balena` metadata folder, including `.balena/qemu-execve` that the CLI adds to images when the `--emulated` flag is given.

@CameronDiver, let me know if you have any concerns about this change, especially the fact that this same code would eventually run in the cloud builders and execute for "balena push" as well.

Reply:
@pdcastro this looks fine to me, odd that it didn't trigger every time though :thinking:

Either way, more than happy to get this merged.

Reply:
I've gathered that this issue was introduced in v11.7.0 - when the metadata code was moved from `resin-builder` to `resin-multibuild`. The error doesn't happen in v11.6.1. I was testing with standalone zip packages for Linux, on an Intel NUC with balenaEngine. Where `SecretRemovalError` applied, however, it would mask the QEMU error.